src/Common/src/Interop/Windows/mincore/Interop.QuerySystemInformation.cs

```diff
@@ -0,0 +1,63 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        private const int DllSearchPathUseSystem32 = 0x800;
+
+        internal enum SystemInformationClass
+        {
+            SystemProcessorPerformanceInformation = 8
+        }
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct SystemProcessorPerformanceInformation
```

Comment:
Should use the same naming scheme as the C++ headers so that it is easy to find and compare, SYSTEM_PROCESSOR_PERFORMANCE_INFORMATION